### ISSUE TITLE
Implement FT and IMU sensors

### ISSUE THREAD

**Issue by PeterBowman** [roboticslab-uc3m/teo-gazebo-models#7]
Luís prepared a modified TEO model including force-torque sensors: https://github.com/roboticslab-uc3m/teo-main/issues/51. Review and upload them here. I recall that the placement was not ideal, i.e. we had to add a dummy link or whatever. Perhaps there is a better way?

Follow-up to https://github.com/roboticslab-uc3m/teo-gazebo-models/issues/1.

**Comment by PeterBowman** [comment]
I'm pasting here the lines added by Gerson, for better visibility.

>the modified TEO model are the following:
>teo URDF: /usr/local/share/iCub/robots/teoSim/ (teoc.urdf)
>teo SDF: /usr/local/share/gazebo/models/teo/ (teo.sdf)
>this directories have to be in the following paths: 
>[teo.zip](https://github.com/roboticslab-uc3m/teo-gazebo-models/files/9817476/teo.zip)
>[teoSim.zip](https://github.com/roboticslab-uc3m/teo-gazebo-models/files/9817477/teoSim.zip)
>If you want to use the Icub's walking-controller you need to modify the walking-controller configuration file, this is found in ( /usr/local/share/ICUBcontrib/robots/teoSim)

**Comment by PeterBowman** [comment]
FT/IMU plugins need to take care of the new wrapping method as previously covered in https://github.com/roboticslab-uc3m/teo-gazebo-models/issues/6 for motor control and cameras. See https://github.com/robotology/gazebo-yarp-plugins/pull/586.

**Comment by PeterBowman** [comment]
I'm following the 1.6 sensor specs defined [here](http://sdformat.org/spec?ver=1.6&elem=sensor). Note I am adjusting the IMU orientation frame via `<pose>` since `<orientation_reference_frame>` with `<localization>CUSTOM</localization>` does not seem to work for me. The IMU local frame is defined in the user manual uploaded to https://github.com/roboticslab-uc3m/datasheets-and-manuals. Our model is MTi-28A53G35 according to the [BOM](https://github.com/roboticslab-uc3m/teo-developer-manual/blob/e89e63a8e1a5d383338b68611c2f57999c9dd5e4/csv/bom.csv).

**Comment by PeterBowman** [comment]
The Gazebo-YARP plugin requires some tweaks in order to overcome the frame orientation issue. In https://github.com/robotology/gazebo-yarp-plugins/pull/639 I'm proposing a new `useWorldReferenceOrientation` configuration option that seeks to match the behavior of the real sensor.

**Comment by PeterBowman** [comment]
Done at https://github.com/roboticslab-uc3m/teo-gazebo-models/commit/ff74a2c1c177f339c297a26f53fde0e3803fa037. I had to split the terminal hand/foot link models in order to place an intermediate fixed joint for the corresponding FT sensor plugin. The masses are just estimates (0.45 kg for the LacqueyFetch, which is somewhat accurate, and 1 kg for the feet) and the inertias have not been adjusted post-split, so those values need to be reviewed at some point.

Notable collateral changes:
* Added non-fixed lacquey fetch and stump worlds.
* The default robot model now lacks grippers. We may add them back easily since I have preserved the original pre-split, unactuated models; just wasn't sure if necessary or worthy.

Note the unmerged commit https://github.com/roboticslab-uc3m/teo-gazebo-models/commit/853263126b1e9c1dac246ab2c41176a01914a984 remains pending review of https://github.com/robotology/gazebo-yarp-plugins/pull/639. (edit: this is no longer necessary, the new default behavior matches our expectations regarding how the IMU works)
